github-runner-manager/src/github_runner_manager/platform/jobmanager_provider.py

```diff
@@ -73,35 +82,32 @@ def get_runner_health(
         Returns:
            The health of the runner in the jobmanager.
         """
-        configuration = jobmanager_client.Configuration(host=self._url)
-        with jobmanager_client.ApiClient(configuration) as api_client:
```

Comment:
We no longer use a context manager. This means, the urllib connection pool will be reused until the reconciliation process dies.

Reply:
Have changed it so we again use context managers per call.